src/components/scopedVariables/SavedVariables.tsx

```diff
@@ -101,10 +122,18 @@ const SavedVariablesView = ({ scopedVariables, setScopedVariables }: SavedVariab
 
                         <CodeEditor value={scopedVariables} mode="yaml" height="100%" readOnly />
                     </div>
-                ) : (
-                    <></>
-                )}
-            </div>
+                </div>
+            ) : (
+                <TableList width={['200px', '429px']} headings={TABLE_LIST_HEADINGS}>
+                    {variablesList?.map((item) => (
+                        <TableItem
+                            key={item.name}
+                            columnsData={[item.name, item.description]}
+                            width={['200px', '429px']}
```

Comment:
Have to avoid hardcoded width. Can pass style to the component either

Reply:
Fixed